src/examples/DataCenter/Input/paraCfgCurrent.txt

```diff
@@ -0,0 +1,20 @@
+ParameterName	ParameterValue
+environmentFileName	DataCenter.xml
+generalizedScenario	1
+indoorSwitch	1
+mobilitySwitch	0
+mobilityType	1
```

Comment:
shouldn't this be 2?

Reply:
We dont have mobility in this scenario. Both Tx and Rx are stationary.